### PR TITLE
add general layout

### PR DESCRIPTION
**Pull request by bohdanburda**

### CODE REVIEW

File: Lanthanum-web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -19,12 +19,15 @@ public class HomeController : Controller
         public HomeController(ILogger<HomeController> logger)
         {
             _logger = logger;
+            Categories = new List<string>
```
Comment (BorysKopytko) [inline Lanthanum-web/Controllers/HomeController.cs:22]: Maybe enum will be better decision.

**Review by BorysKopytko — CHANGES_REQUESTED** [review]
Please think about changes that were suggested in comments.

File: Lanthanum-web/Controllers/HomeController.cs (hunk 1 above)
Comment (bohdanburda) [reply Lanthanum-web/Controllers/HomeController.cs:22]: But then there are problems with adding/removing categories by admin

Comment (BorysKopytko) [reply Lanthanum-web/Controllers/HomeController.cs:22]: Ok, I agree with you. You have my approve.

**Review by BorysKopytko — APPROVED** [review]
Everything is OK.

**Review by oleksandrmanetskyi — APPROVED** [review]
👍

**Pull request closed without merging**